shared/css/cookie-banner.scss

```diff
@@ -4,8 +4,9 @@
 #cookie-banner {
   position: fixed;
   bottom: 0px;
+  right: 0px;
+  left: 0px;
   overflow: hidden;
-  width: 100%;
```

Comment:
This change should be fine but we should be careful given that the cookie banner appears on almost every page of our site.  We also suppress its display for UI tests (so that we don't interfere with the existing suite) so it's not getting as much coverage as one might think.  cc @tanyaparker and @Erin007 just so they know about this change in case any side effects appear, too.

Reply:
Wondering if it's safer to pre-flight this change in a standalone PR?

Reply:
Sounds like we're relying on manual test coverage. Happy to repeat whatever manual test steps we used before. I've verified it on Chrome macOS and Safari iOS. It looks the same as before.